### PR TITLE
Kyc kyb reports

### PR DESCRIPTION
**Pull request by deanrecord**

### CODE REVIEW

**Comment by deanrecord** [comment]
Screenshot 2020-07-15 at 15 29 18
Screenshot 2020-07-15 at 15 29 26

File: source/includes/_investors.md [hunk 1]
```diff
@@ -552,7 +552,7 @@ Updates the investor's classification.
 
 ```http
 
-GET /platformApi/investors/{clientId}/kyc HTTP/1.1
+GET /platformApi/investors/GOJI-1234/kyc HTTP/1.1
```
Comment (badgerwithagun) [inline source/includes/_investors.md:555]: Looks like the `{paramName}` pattern is used a lot elsewhere, e.g.

* https://docs.api.goji.investments/#payments-investors-post-platformapi-investors-clientid-accounts-accounttype-withdrawal
* https://docs.api.goji.investments/#bond-management-post-platformapi-investors-clientid-accounts-accounttype-investments

Worth sticking with it?  Or are we moving towards the example approach instead?  Or did I just start doing this inadvertently in mine and set a bad example?

File: source/includes/_investors.md [hunk 2]
```diff
@@ -581,29 +581,29 @@ Content-Type: application/json
...
-| lastFailure                                         | ref    |                                                                                                                                                                                     |
-| lastFailure.failureDate                             | string | Date of rejection                                                                                                                                                                   |
-| lastFailure.rejectedDocuments                       | array  | The rejected documents.                                                                                                                                                             |
+| clientId                                               | string | The clientId of the investor                                                                                                                                                        |
```
Comment (badgerwithagun) [inline source/includes/_investors.md:593]: Prettier reformatting?

File: source/includes/_investors.md [hunk 3]
```diff
@@ -581,29 +581,29 @@ Content-Type: application/json
...
+| {}.lastFailure                                         | ref    |                                                                                                                                                                                     |
+| {}.lastFailure.failureDate                             | string | Date of rejection                                                                                                                                                                   |
+| {}.lastFailure.rejectedDocuments                       | array  | The rejected documents.
+                                                                                                                                                         |
 ## `GET /platformApi/investors/{clientId}/kyb`
 
 ```http
 
-GET /platformApi/investors/{clientId}/kyb HTTP/1.1
+GET /platformApi/investors/GOJI-1234/kyb HTTP/1.1
```
Comment (badgerwithagun) [inline source/includes/_investors.md:606]: DItto

File: source/includes/_investors.md (hunk 1 above)
Comment (deanrecord) [reply source/includes/_investors.md:555]: I went with your example which I personally think is better that we show real examples.

File: source/includes/_investors.md (hunk 2 above)
Comment (deanrecord) [reply source/includes/_investors.md:593]: Fat fingers probably

File: source/includes/_investors.md [hunk 4]
```diff
@@ -616,16 +616,35 @@ Authorization: Basic ...
 HTTP/1.1 200 OK
 Content-Type: application/json
 
-{
-  "status" : "DOCUMENTS_REQUIRED"
-}
+    "documentsRequired" : [ "PROOF_OF_ADDRESS", "PROOF_OF_IDENTITY" ],
```
Comment (badgerwithagun) [inline source/includes/_investors.md:619]: Did we lose the opening and closing braces here?

Comment (deanrecord) [reply source/includes/_investors.md:619]: The payload in the docs was completley incorrect. I've updated it to be actually whats passed back

File: source/includes/_investors.md (hunk 1 above)
Comment (badgerwithagun) [reply source/includes/_investors.md:555]: I can't see anywhere that explains what `GOJI-1234` represents though (i.e. this is the investor client id)

File: source/includes/_investors.md (hunk 4 above)
Comment (badgerwithagun) [reply source/includes/_investors.md:619]: So we don't pass back valid JSON?

Comment (badgerwithagun) [reply source/includes/_investors.md:619]: There's no `{ ... }` around the entire example

Comment (deanrecord) [reply source/includes/_investors.md:619]: Oh sorry misunderstood I thought you meant under status. My error

**Comment by deanrecord** [comment]
I've put the {clientId} back in for clarity. No issues other than a few merge conflicts. Thanks

**Pull request merged**